sdk/src/test/java/com/google/cloud/dataflow/sdk/runners/inprocess/InProcessPipelineRunnerTest.java

```diff
@@ -88,6 +91,50 @@ public String apply(KV<String, Long> input) {
     result.awaitCompletion();
   }
 
+  private static AtomicInteger changed;
```

Comment:
It can't be because if it is it gets cloned with the DoFn and we don't get the correct output.